web-ui/src/main/resources/catalog/components/usersearches/partials/portalswitcher.html

```diff
@@ -1,7 +1,7 @@
 <li
   class="dropdown gn-clear-xs"
   role="menuitem"
-  data-ng-if="showPortalSwitcher && ((portals | filter: {listableInHeaderSelector: true}).length > 0 || nodeId != 'srv')"
+  data-ng-show="showPortalSwitcher && ((portals | filter: {listableInHeaderSelector: true}).length > 0 || nodeId != 'srv')"
```

Comment:
I'm not sure about this change.

Reply:
With `ng-if` it was not working probably because the directive was also having a `ng-if` I think in https://github.com/geonetwork/core-geonetwork/pull/7138/files#diff-4b283abc0f27d9bc4eab10794894d6654513d53350d3d721dcb98b3a7749f4a5R44 Not sure if there is a better way to handle that.